docker/Gemfile

```diff
@@ -1,9 +1,13 @@
 source 'https://rubygems.org'
 
 gem 'puma'
-gem 'txgh', '~> 6.0'
-gem 'txgh-queue', '~> 2.0'
-gem 'txgh-server', '~> 3.0'
+gem 'txgh', path: '../txgh'
```

Comment:
Can we switch these back to published gems?

Reply:
I think we should finish the review and then publish & set published versions. Also who's able to push the versions to rubygems? I thought you have the credentials?